jetty-util/src/test/java/org/eclipse/jetty/util/thread/QueuedThreadPoolTest.java

```diff
@@ -382,24 +383,24 @@ public void testLifeCycleStop() throws Exception
         // Wait until the first 2 start running
         waitForThreads(tp,2);
         waitForIdle(tp,0);
+        assertTrue(job0._run.await(200, TimeUnit.MILLISECONDS));
+        assertTrue(job1._run.await(200, TimeUnit.MILLISECONDS));
 
         // Queue should be empty after thread pool is stopped
         tp.stop();
         assertThat(tp.getQueue().size(), is(0));
 
         // First 2 jobs closed by InterruptedException
-        assertThat(job0._stopped.await(200, TimeUnit.MILLISECONDS), is(true));
-        assertThat(job1._stopped.await(200, TimeUnit.MILLISECONDS), is(true));
+        assertTrue(job0._stopped.await(200, TimeUnit.MILLISECONDS));
+        assertTrue(job1._stopped.await(200, TimeUnit.MILLISECONDS));
 
         // Verify RunningJobs in the queue have not been run
-        assertThat(job2._run.await(200, TimeUnit.MILLISECONDS), is(false));
-        assertThat(job4._run.await(200, TimeUnit.MILLISECONDS), is(false));
+        assertFalse(job2._run.await(200, TimeUnit.MILLISECONDS));
+        assertFalse(job4._run.await(200, TimeUnit.MILLISECONDS));
 
         // Verify ClosableJobs have not been run but have been closed
-        assertThat(job4._run.await(200, TimeUnit.MILLISECONDS), is(false));
-        assertThat(job3._closed.await(200, TimeUnit.MILLISECONDS), is(true));
-
-        tp.stop();
+        assertFalse(job3._run.await(200, TimeUnit.MILLISECONDS));
```

Comment:
this changed from `_job4` to `_job3`, was that intentional?

Reply:
yes, otherwise it was just a duplicate of the line above, it should be testing `job3` was not run